docs/blog/2018-2-12-case-study-mike-johnston/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Case Study: Mike Johnston for Colorado"
+date: 2018-02-12
+author: Brandon Konkle
+image: mike-johnston-homepage.png
+---
+
+### How we used React, Headless WordPress, GraphQL & Gatsby to build Mike’s new online platform
+
+[Mike Johnston](https://www.mikejohnstonforcolorado.com) is a Colorado gubernatorial candidate with an ambitious vision. When I heard about his ideas for an online platform to truly engage with his constituents and give them a real voice, I was excited about the opportunity to make his ideas into reality!
+
+While we’re still a few phases away from seeing Mike’s ultimate vision come to fruition, we’ve got the foundations in place and we’re putting the finishing touches on a gorgeous new interface for Mike’s homepage. Check it out at [https://www.mikejohnstonforcolorado.com](https://www.mikejohnstonforcolorado.com)!
+
+## Laying the Foundation for an Extensible Future
+
+Our goal was to provide a solid architecture to build a long-term future on top of. In later phases the site will become a full-fledged web application with an organizing and fundraising platform built-in, and the goal beyond that is to create an innovative “open source policy” platform to involve citizens in the policy-making process like never before!
+
+To execute on this effectively, I prioritized a few architectural goals:
```

Comment:
Use "we" instead of "I" -- need to maintain a consistent pluralization of subject and "we" reads better